CHANGELOG.md

```diff
@@ -7,11 +7,14 @@
   - Features Added
   - Bugs Fixed
 
-## 2020-03-02
+## 2020-03-03
+
+    - Add curaDAI
```

Comment:
why this dao need special support in alchemy ?

Reply:
@orenyodfat It is a token , not a DAO.